kornia/feature/lightglue_onnx/lightglue.py

```diff
@@ -42,25 +43,35 @@ class OnnxLightGlue:
 
     required_data_keys: ClassVar[list[str]] = ["image0", "image1"]
 
-    def __init__(self, weights: str = "disk_fp16", device: Device = None) -> None:
+    def __init__(self, weights: str | None = None, device: Device = "cpu") -> None:
         KORNIA_CHECK(ort is not None, "onnxruntime is not installed.")
         KORNIA_CHECK(np is not None, "numpy is not installed.")
 
         if device is None:
-            device = torch.device("cuda")
+            device = torch.device("cpu")
```

Comment:
i believe you do not need the check if the device is None here -- just use `device = torch.device(device)` should works

Reply:
Sorry, I was under the impression that a user could input `None` since `Device` is defined as:
https://github.com/kornia/kornia/blob/d0eb5cddfc9c2e5ce3a2727de8949f33f5a57dac/kornia/core/_backend.py#L42